yarn deploy errors with Cannot find module 'postcss'

After fixing other issues, I get this message

$ tailwindcss build ./src/styles/index.css -o ./public/styles/styles.css
[ssh ethanstrominger@192.168.1.177|stderr] node:internal/modules/cjs/loader:936
[ssh ethanstrominger@192.168.1.177|stderr] throw err;
[ssh ethanstrominger@192.168.1.177|stderr] ^
Error: Cannot find module 'postcss'

Additional info that may help:
To install postcss, I ran yarn add postcss, then tried yarn global add postcss
I needed to add statement PATH=$PATH:/usr/local/bin into work.sh or the script did not find yarn
Script prompts for password at these steps:
- scp -i /Users/ethanstrominger/.ssh/id_rsa /Users/ethanstrominger/projects/mobtime/deploy/work.sh ethanstrominger@192.168.1.177:/tmp/deploy_mobtime_1641761587707.sh
- Running `ssh -i /Users/ethanstrominger/.ssh/id_rsa ethanstrominger@192.168.1.177 bash /tmp/deploy_mobtime_1641761587707.sh`..
- ssh -i /Users/ethanstrominger/.ssh/id_rsa ethanstrominger@192.168.1.177 bash /tmp/deploy_mobtime_1641760549994.sh

--- a/deploy/work.sh.example
+++ b/deploy/work.sh.example
@@ -23,8 +23,8 @@ rm -rf $APP_DIR/next/.git
 
 echo Installing dependencies...
 cd $APP_DIR/next
-yarn --prod --non-interactive
-yarn tailwind
+npm ci --omit=dev --ignore-scripts
+npm run tailwind
 
 echo Swapping fresh deploy with current code...
 rm -rf $APP_DIR/previous

--- a/package.json
+++ b/package.json
@@ -24,6 +24,7 @@
     "ferp": "^2.1.2",
     "helmet": "^4.6.0",
     "nodemon": "^2.0.15",
+    "postcss": "^8.4.5",
     "prettier-eslint-cli": "^5.0.1",
     "redis": "^4.0.1",
     "tailwindcss": "^3.0.15",

--- a/package.json
+++ b/package.json
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "dev": "nodemon --ignore public ./index.js",
-    "tailwind": "tailwindcss build ./src/styles/index.css -o ./public/styles/styles.css",
+    "tailwind": "tailwindcss build -i ./src/styles/index.css -o ./public/styles/styles.css",
     "start": "node ./index.js",
     "lint": "prettier-eslint --write \"./src/**/*.js\" \"./public/**/*.js\"",
     "deploy": "node ./deploy/index.js",
@@ -18,7 +18,6 @@
     "prepare": "husky install"
   },
   "dependencies": {
-    "autoprefixer": "^10.0.1",
     "body-parser": "^1.19.0",
     "esm": "^3.2.25",
     "express": "^4.17.2",
@@ -27,7 +26,7 @@
     "nodemon": "^2.0.15",
     "prettier-eslint-cli": "^5.0.1",
     "redis": "^4.0.1",
-    "tailwindcss": "^3.0.8",
+    "tailwindcss": "^3.0.15",
     "ws": "^8.4.0"
   },
   "devDependencies": {

--- a/tailwind.config.js
+++ b/tailwind.config.js
@@ -1,9 +1,5 @@
 module.exports = {
-  future: {
-    // removeDeprecatedGapUtilities: true,
-    // purgeLayersByDefault: true,
-  },
-  purge: [
+  content: [
     './public/**/*.html',
     './public/**/*.js',
     './src/**/*.html',

